CMakeLists.txt

```diff
@@ -150,6 +150,8 @@ add_library(geom STATIC
   LEGO1/lego/sources/geom/legomesh.cpp
   LEGO1/lego/sources/geom/legosphere.cpp
   LEGO1/lego/sources/geom/legovertex.cpp
+  "LEGO1/lego/sources/misc/legoweedge.cpp"
```

Comment:
These are part of `geom` according to the asserts in the beta

Reply:
Oh, there's references to the .h file, I missed that.  The cpp asserts didn't have a path, so I was guessing based on binary location.